client.go

```diff
@@ -68,25 +73,20 @@ func getDockerClient() DockerClient {
 	return dockerClient
 }
 
-// Run the given command in a container based on the given image. The given
-// image string is just the ID of said image. It returns true if the command
-// was successful, false otherwise.
-func runCommandInContainer(img string, cmd []string) bool {
+// Looks for the specified command inside of a Docker image.
+// The given image string is just the ID of said image.
+// It returns true if the command was successful, false otherwise.
+func checkCommandInImage(img string, cmd string) bool {
```

Comment:
The array is requested by `createContainer` because Docker needs to know also the parameters to use.

In this context (`checkCommandInImage`) we are just looking for the presence of executable `foo` inside of a Docker image. How we figure this out does not require any parameter. That's why I decided to not use an array.